geoindicators/src/main/groovy/org/orbisgis/RsuIndicators.groovy

```diff
@@ -619,4 +615,164 @@ static IProcess rsuEffectiveTerrainRoughnessHeight() {
 
                 [outputTableName: outputTableName]
             }
+    )}
+
+/** Performs operations on the linear of road within the RSU scale objects. Note that when a road is located at
+ * the boundary of two RSU, it is arbitrarily attributed to the RSU having the lowest ID in order to not
+ * duplicate the corresponding road.
+ *
+ * @param datasource A connexion to a database (H2GIS, PostGIS, ...) where are stored the input Table and in which
+ * the resulting database will be stored
+ * @param rsuTable the name of the input ITable where are stored the RSU geometries
+ * @param roadTable the name of the input ITable where are stored the road geometries
+ * @param operations the name of the geospatial variables to calculated using the linear of road:
+ *      -> "rsu_road_direction_distribution": The direction of each segment of road is calculated. The percentage of linear of road
+ *  * in each range of direction is then calculated (a range is defined - default 30°) for directions
+ *  * included in [0, 180[°).
+ *      -> "rsu_linear_road_density": linear of road within an area divided by the rsu_area.
+ * @param levelConsiderated the indicators can be calculated independantly for each level (0 indicates that the object
+ * is on the ground. 1 to 4 indicates that the object is in the air. -4 to -1 value indicates that the object is
+ * underground) or not (null). Default [0, 1]
+ * @param prefixName String use as prefix to name the output table
+ * @param angleRangeSize the range size (in °) of each interval angle used to calculate the distribution of road per
+ * direction (should be a divisor of 180 - default 30°)
+ *
+ * @return outputTableName Table name in which the rsu id and their corresponding indicator value are stored
+ *
+ * @author Jérémy Bernard
+ */
+static IProcess rsuLinearRoadOperations() {
+    return processFactory.create(
+            "Operations on the linear of road",
+            [rsuTable: String, roadTable: String, operations: String[], prefixName: String, angleRangeSize: int,
+             levelConsiderated: int[], datasource: JdbcDataSource],
+            [outputTableName : String],
+            { rsuTable, roadTable, operations, prefixName, angleRangeSize = 30, levelConsiderated = [0, 1],
```

Comment:
default prefixName ?

Reply:
This question is still waiting for an answer in issue #51: how should we use the prefixName ? In my opinion, by default I would use prefixName = "" in order to only have the outputTableName. Then if the user wants to differentiate cases, he can adjust using a prefix.